staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -149,9 +149,11 @@ func (f *FieldManager) Update(liveObj, newObj runtime.Object, manager string) (r
 // Apply is used when server-side apply is called, as it merges the
 // object and update the managed fields.
 func (f *FieldManager) Apply(liveObj runtime.Object, patch []byte, fieldManager string, force bool) (runtime.Object, error) {
-	// If the object doesn't have metadata, apply isn't allowed.
-	if _, err := meta.Accessor(liveObj); err != nil {
+	// If the object doesn't have metadata or managed fields is not empty, apply isn't allowed.
+	if objMeta, err := meta.Accessor(liveObj); err != nil {
```

Comment:
`liveObj` is the object as it is on the server, the server is allowed to have managed fields!

Reply:
So how can I check for managed fields? Cant the patch take many forms?

Reply:
Ignore my last comment, I found my mistake